.github/workflows/build.yml

```diff
@@ -43,179 +37,307 @@ jobs:
+
+      - name: Restore packages
+        run: nuget restore
+
+      - name: Build for F-Droid
         run: |
-          node --version
-          npm --version
-          git --version
-          Write-Output "GitHub ref: $env:GITHUB_REF"
-          Write-Output "GitHub event: $env:GITHUB_EVENT"
+          $configuration = "FDroid";
+
+          Write-Output "########################################"
+          Write-Output "##### Build $configuration Configuration"
+          Write-Output "########################################"
+
+          msbuild "$($env:GITHUB_WORKSPACE + "/src/Android/Android.csproj")" "/p:Configuration=$configuration"
         shell: pwsh
+
+      - name: Sign for F-Droid
         env:
-          GITHUB_REF: ${{ github.ref }}
-          GITHUB_EVENT: ${{ github.event_name }}
+          FDROID_KEYSTORE_PASSWORD: ${{ secrets.FDROID_KEYSTORE_PASSWORD }}
+        run: |
+          Write-Output "########################################"
+          Write-Output "##### Sign FDroid Configuration"
+          Write-Output "########################################"
 
-      - name: Checkout repo
-        if: github.event_name == 'release'
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+          msbuild "$($env:GITHUB_WORKSPACE + "/src/Android/Android.csproj")" `
+            "/t:SignAndroidPackage" "/p:Configuration=FDroid" "/p:AndroidKeyStore=true" `
+            "/p:AndroidSigningKeyAlias=bitwarden" "/p:AndroidSigningKeyPass=$($env:FDROID_KEYSTORE_PASSWORD)" `
+            "/p:AndroidSigningKeyStore=$("app_fdroid-keystore.jks")" `
+            "/p:AndroidSigningStorePass=$($env:FDROID_KEYSTORE_PASSWORD)" "/v:quiet"
 
-      - name: Install Node dependencies
-        if: github.event_name == 'release'
-        run: npm install
+          Write-Output "########################################"
+          Write-Output "##### Copy FDroid apk to project root"
+          Write-Output "########################################"
 
-      - name: Decrypt secrets
-        if: github.event_name == 'release'
-        run: |
-          ./.github/scripts/decrypt-secret.ps1 -filename store_fdroid-keystore.jks.gpg `
-            -output ./store/fdroid/keystore.jks
+          $signedApkPath = $($env:GITHUB_WORKSPACE + "/src/Android/bin/FDroid/com.x8bit.bitwarden-Signed.apk");
+          $signedApkDestPath = $($env:GITHUB_WORKSPACE + "/com.x8bit.bitwarden-fdroid.apk");
+
+          Copy-Item $signedApkPath $signedApkDestPath
         shell: pwsh
-        env:
-          DECRYPT_FILE_PASSWORD: ${{ secrets.DECRYPT_FILE_PASSWORD }}
 
-      - name: Compile for F-Droid Store
-        if: github.event_name == 'release'
+      - name: Upload F-Droid .apk artifact
+        uses: actions/upload-artifact@ee69f02b3dfdecd58bb31b4d133da38ba6fe3700  # v2.2.4
+        with:
+          name: com.x8bit.bitwarden-fdroid.apk
+          path: ./com.x8bit.bitwarden-fdroid.apk
+
+      - name: Check if RC branch exists
```

Comment:
Is this step needed since it is the last in the job?

Reply:
Whoops, good catch.  I removed the step.